packages/qwik/src/core/render/jsx/types/jsx-generated.ts

```diff
@@ -1,10 +1,28 @@
+import * as CSS from 'csstype';
 import type { Signal } from '../../../state/signal';
 import type { DOMAttributes, ClassList } from './jsx-qwik-attributes';
 interface HTMLWebViewElement extends HTMLElement {}
 export type Booleanish = boolean | `${boolean}`;
 export type Size = number | string;
 export type Numberish = number | `${number}`;
 
+/**
+ * @public
+ */
+export interface CSSProperties
```

Comment:
I think it makes sense to export it so library authors and qwik users can reuse it.